classes/PodsI18n.php

```diff
@@ -256,12 +256,12 @@ public function get_current_language( $args = array() ) {
 	 */
 	public function get_current_language_data( $args = array() ) {
 
-		$args = wp_parse_args(
-			$args, array(
-				'refresh' => false,
-			)
+		$defaults = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 47b8fd00226044acb39760c425e9c29c</sub>